src/console/autoSwitcher.ts

```diff
@@ -51,17 +51,21 @@ export class AutoSwitcher extends EventEmitter {
   public async connect() {
     if (this.obsIP && this.obsSourceName) {
       // if you send a password when authentication is disabled, OBS will still connect
-      await this.obs.connect(
-        {
+      try {
+        await this.obs.connect({
           address: this.obsIP,
           password: this.obsPassword,
-        },
-        (err) => {
-          if (err) {
-            this.emit(MirrorEvent.ERROR, err);
-          }
-        },
-      );
+        });
+      } catch (err) {
+        if (err) {
```

Comment:
Is it possible for an error to be thrown with `err` being falsey? Cuz if it isn't then we shouldn't need to check for `err` existing. If it is possible for it to be falsy, can we reverse the check to save an indentation level?

```suggestion
        if (!err) {
          return;
        }
```

Reply:
yeah this shouldn't even need a check